environments/backfillEvents.toml

```diff
@@ -3,7 +3,35 @@
     name     = "transformerExporter"
     save     = true
     schema   = "maker"
-    transformerNames = []
+    transformerNames = ["clip_kick", "clip_take", "clip_redo", "clip_yank"]
+    [exporter.clip_kick]
+        contracts = ["MCD_CLIP_AAVE_A_1_6_0", "MCD_CLIP_BAL_A_1_6_0", "MCD_CLIP_BAT_A_1_6_0", "MCD_CLIP_COMP_A_1_6_0", "MCD_CLIP_ETH_A_1_5_0", "MCD_CLIP_ETH_B_1_5_0", "MCD_CLIP_ETH_C_1_5_0", "MCD_CLIP_KNC_A_1_6_0", "MCD_CLIP_LRC_A_1_6_0", "MCD_CLIP_LINK_A_1_3_0", "MCD_CLIP_MANA_A_1_6_0", "MCD_CLIP_RENBTC_A_1_6_0", "MCD_CLIP_UNI_A_1_6_0", "MCD_CLIP_WBTC_A_1_5_0", "MCD_CLIP_YFI_A_1_5_0", "MCD_CLIP_ZRX_A_1_6_0", "MCD_CLIP_UNIV2DAIETH_A_1_8_0", "MCD_CLIP_UNIV2USDCETH_A_1_8_0", "MCD_CLIP_UNIV2ETHUSDT_A_1_8_0", "MCD_CLIP_UNIV2WBTCDAI_A_1_8_0", "MCD_CLIP_UNIV2WBTCETH_A_1_8_0", "MCD_CLIP_UNIV2LINKETH_A_1_8_0", "MCD_CLIP_UNIV2UNIETH_A_1_8_0", "MCD_CLIP_UNIV2AAVEETH_A_1_8_0", "MCD_CLIP_UNIV2DAIUSDT_A_1_8_0"]
```

Comment:
Do you need the 1.5 and 1.6 contracts in here, or just the 1.8 contracts that you added below?